http/jsonapi/runtime/standart_params.go

```diff
@@ -63,46 +60,86 @@ func (m *MapMapper) Map(value string) (string, bool) {
 	return val, isValid
 }
 
-// PaginationFromRequest extracts pagination query parameter and  returns a function that adds the pagination to a query
-func PaginationFromRequest(r *http.Request) (QueryOption, error) {
+// UrlQueryParameters contains all information that are needed for pagination, sorting and filtering.
```

Comment:
either pluralize information or use "that is needed", or maybe my english is not good enough :D

Reply:
I looked it up: There is no plural of "information" :)